### ISSUE TITLE
Review Ticket for Sonification Tutorial

### ISSUE THREAD

**Issue by ianmilligan1** [programminghistorian/ph-submissions#4]
This ticket will host peer review comments for @shawngraham's tutorial ["Algorithmic Listening, (or, a gentle introduction to the sonification of historical data)".](http://programminghistorian.github.io/ph-submissions/lessons/sonification).

Stay tuned.

**Comment by ianmilligan1** [comment]
I've reached out to peer reviewers and have contacted two. I've apologized privately to Shawn, but have been on the road for ten days and struck by illness. I hope the next few stages can go more smoothly.

I will act as editor for the review process. My role is to solicit two reviews from the community and to manage the discussions, which should be held here on this forum. I have already read through the lesson and provided feedback, to which the author has responded.

Members of the wider community are also invited to offer constructive feedback which should post to this message thread, but they are asked to first read our Reviewer Guidelines (http://programminghistorian.org/reviewer-guidelines) and to adhere to our anti-harassment policy (below). We ask that all reviews stop after the second formal review has been submitted so that the author can focus on any revisions. I will make an announcement on this thread when that has occurred.

I will endeavour to keep the conversation open here on Github. If anyone feels the need to discuss anything privately, you are welcome to email me. You can always turn to @miriamposner if you feel there's a need for an ombudsperson to step in.

**Anti-Harassment Policy**

> This is a statement of the Programming Historian's principles and sets expectations for the tone and style of all correspondence between reviewers, authors, editors, and contributors to our public forums.
> 
> The Programming Historian is dedicated to providing an open scholarly environment that offers community participants the freedom to thoroughly scrutize ideas, to ask questions, make suggestions, or to requests for clarification, but also provides a harassment-free space for all contributors to the project, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion, or technical experience. We do not tolerate harassment or ad hominem attacks of community participants in any form. Participants violating these rules may be expelled from the community at the discretion of the editorial board. If anyone witnesses or feels they have been the victim of the above described activity, please contact our ombudspersons (Ian Milligan or Miriam Posner - http://programminghistorian.org/project-team). Thank you for helping us to create a safe space.

**Comment by shawngraham** [comment]
I'll just note here that musicalgorithms just updated their site. So the data input for the latest version - 3.5 - is a bit different than what I wrote. So I'll direct the reader to use version 3.0 which is at http://musicalgorithms.org/3.0/index.html

**Comment by shawngraham** [comment]
It occurs to me that perhaps sonification is a step too far for most readers of programming historian, ie, just not something for which there is any demand. Should we just shelve this?

**Comment by ianmilligan1** [comment]
I think it's valuable, having given this a first read through and enjoyed it. I've got two peer reviewers currently assigned who are quite keen too. 

Why don't we see what they think? My own take is that there's clear value in pushing the boundaries here and encouraging people to think alternatively about their sources.

**Comment by shawngraham** [comment]
alrighty then!

**Comment by JD2Veitch** [comment]
I agree with the importance of your tutorial, Shawn, despite it being 'out-there' for some readers. I particularly liked the background you start with and placing sonification in its context. The relative newness of sonification means that the tutorial sets a foundation for further exploration by users.

Overall, I found the tutorial clear and insightful. I am not as knowledgeable about python so, I had to do some extra reading/playing to get a sense of it. There is a bit of a jump in terms of skills and familiarity between Musicalgorithms and MIDITime. While Musicalgorithms is straightforward and easy to navigate, MIDITime required some prior knowledge (hence, my reading up on python). I assume that many of the Programing Historian folks will be more familiar with python, then I am. 

I wonder if there needs to be any intro to the music software (Garageband, Logic, Sibielius, Pro Tools, etc; I give away my Mac usage listing these programs...). I know that most are paid programs and, therefore, have there own intros, but maybe some links? I am far more knowledgeable of the sound recording tech, than programing and coding, and know that some of the programs have their own learning curves. I wonder how many people dig into the music programs and know their ins and outs. This might be another aspect of pushing readers to explore on their own and could add to the diversity of applications...

I do think there is a lot of potential in exploring sonification and the tutorial offers a way into some of those possibilities. I found the workflow clear and easy to follow. The creative aspects of sonification make the payoff both interesting and, to some, hard to grasp. There is a sense that certain people will gravitate towards sonification because it allows for alternative approaches to sources, data, etc. The tutorial shows many of the creative potentials sonification allows for.

**Comment by ianmilligan1** [comment]
Thanks so much, @JD2Veitch, for your insightful and engaged review of @shawngraham's review. I think this gives a good pathway forward.

@shawngraham: we're still waiting on one more review, which I'm expecting soon. I will then synthesize comments, provide editorial guidance, and we'll be well on our way to hopefully seeing your lesson in _Programming Historian_ soon.

**Comment by TimCompeau** [comment]
Hi everyone! Sorry for the tardiness of my review. I am a “just in the nick of time” kind of reviewer.

My first introduction to the idea of hearing the past or archaeoacoustics was from Shawn and his work with Stuart Eve et al, so it is only fitting I get my first taste of sonification from Shawn, as well. So, this review comes from someone who is (or was) completely unfamiliar with even the basic concept of sonification. Lucky for me you lead the reader into the subject very gently. 

I fully agree with Jeff’s assessment. I found this very easy to follow and worked through it at a leisurely pace. I don’t have many substantial comments about the step-by-step lesson you have laid out. Your knowledge and teaching ability come through, and I enjoyed the process. (There are a couple dead links, but they didn’t cause any serious problems). As Jeff pointed out the learning curve does get steeper as you move along, but that is in keeping with _The Programming Historians_’ lessons.

In an earlier comment, Shawn wondered if this was “a step too far.” I agree that this won’t be a lesson for everyone. Many historians are not going to get why they should do this, or see how this could be of use in their research. Maybe that is a part of the ocularcentrism of our discipline! But I think the DH community is pretty open to new ideas. I wonder if you should really drive home the idea of the benefits and possibilities of experimentation for its own sake and the potential of discovery in your introduction. Your link and quotation in the conclusion from Trevor Owens really sums up the idea nicely. Perhaps there is room in the _Programming Historian_ for a whole section or blog-style venue for this type of investigation. 

So, in summary, I think the lesson provides an excellent introduction to the idea of sonifying data for historians already familiar with the other lessons and I wouldn’t change too much of the practice elements. This is a new idea for historians, and might meet with some resistance. But these lessons could equip someone with the tools to make new and interesting discoveries. The bizarre sensation of hearing the words to that Smash Mouth song when they weren’t really there convinced me that there is a lot out there to discover, and this lesson could be a great way to introduce new research possibilities.

**Comment by ianmilligan1** [comment]
Fantastic, thanks @TimCompeau! 

Now that we have these reviews in, I'll close this ticket to public comment (waves magic hands). More importantly, let me review these, the original manuscript, and make some synthesizing comments for @shawngraham.

**Comment by ianmilligan1** [comment]
Thanks again for @JD2Veitch and @TimCompeau for their comments, and for @shawngraham for writing this tutorial.

I see some common themes in the two lesson, which jibe well with my own experiences. Both reviewers thought that this lesson offered lots of potential for sonification, and is a good introduction to this new and emerging area.

A few suggestions then:
- @JD2Veitch suggested maybe providing some more information around Python when using MIDITime. I'll leave this in your court, @shawngraham, whether you want this to be links to other _Programming Historian_ lessons or some more information.
- @JD2Veitch suggested introduction to music programs.. I think some links would be handy.
- @TimCompeau encouraged "driving home the benefits and possibilities of experimentation for its own sake and the potential of discovery in your introduction" - I agree, and know you've done this elsewhere.

So I'll let you take the reviews to heart. Let me know when you've made the changes and we can begin to move forward.

**Comment by shawngraham** [comment]
Hi everyone – thank you all for the comments. I believe I’ve now addressed these, and so I think this tutorial is ready to go.

**Comment by ianmilligan1** [comment]
Thanks @shawngraham. Some tiny little things:
- On line 123, you have `**Some screenshots would help with the following stages**` still in there. I assume you don't want to add screenshots? In any case, we should remove that line. I will let you do so.
- We have the terms at the bottom. Do you want to make internal links from their first mention – i.e. when you first mention "MP3" should there be an internal link to it?
- Same with the in-text citations?
- I made your video rather than clickable embeddable.

Anyways, let me know your thoughts on these.

**Comment by shawngraham** [comment]
- Ah, that line 123 is superfluous, as there are in fact screenshots in that section. I’ve removed that line.
- Y’know, I’ve never made internal links before. I can do that.
- Cheers re the video.

Will ping you shortly once I’ve put in the internal links.

**Comment by shawngraham** [comment]
Ok, I’ve addressed all these things.

**Comment by ianmilligan1** [comment]
Great! We're beginning the migration process. I'll get you to review proofs and then we will create a little image for the main page. Am closing this now.
